src/coreclr/hosts/inc/coreclrhost.h

```diff
@@ -27,7 +27,8 @@ CORECLR_HOSTING_API(coreclr_initialize,
 
 CORECLR_HOSTING_API(coreclr_shutdown,
             void* hostHandle,
-            unsigned int domainId);
+            unsigned int domainId,
```

Comment:
@janvorli  Should I do the same change on `ICLRRuntimeHost::UnloadAppDomain` (keep the old one, add a new `ICLRRuntimeHost::UnloadAppDomain2`)?

BTW, can we just set the default value of latchedExitCode to NULL? That's what I did on  `ICLRRuntimeHost::UnloadAppDomain`